resspect/lightcurves_utils.py

```diff
@@ -0,0 +1,160 @@
+"""
+    utils for fit_lightcurves methods
+"""
+
+import io
+import tarfile
+from typing import AnyStr
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+
+from resspect.snana_fits_to_pd import read_fits
+
+SNPCC_LC_MAPPINGS = {
+    "snii": ['2', '3', '4', '12', '15', '17', '19', '20', '21', '24', '25',
```

Comment:
given how you use this in `get_snpcc_sntype(..)`, I would make each value a `set` rather than a `list`.